config/env.production

```diff
@@ -112,6 +112,24 @@ POSTS_URL=https://api.telescope.cdot.systems/v1/posts
 MOCK_REDIS=
 
 
+################################################################################
+# User Service
+################################################################################
+
```

Comment:
You also need to specify the values for `USER_PORT` and `USER_URL` in all 3 files.  See the other services in these files for clues about the values.